xt_imager.py

```diff
@@ -33,6 +33,12 @@ def main():
         default='/dev/ttyUSB0',
         help='Serial console to use')
 
+    parser.add_argument(
+        '-b',
+        '--baud',
+        default=921600,
```

Comment:
I am not sure that this is a good idea as this will break existing setups.

On other hand, it is still in early development stages... So it is up to you.